test/unit/specs/components/common/TmConnectedNetwork.spec.js

```diff
@@ -2,19 +2,24 @@ import TmConnectedNetwork from "common/TmConnectedNetwork"
 import setup from "../../../helpers/vuex-setup"
 
 describe(`TmConnectedNetwork`, () => {
-  let wrapper, store, instance
+  let wrapper, store
```

Comment:
That's a nice way  👍I would only switch the 
`let instance` to `const instance`